packages/react-native/scripts/packager.sh

```diff
@@ -14,11 +14,6 @@ PROJECT_ROOT=${PROJECT_ROOT:-"$THIS_DIR/../../.."}
 # shellcheck disable=SC1090
 source "${THIS_DIR}/node-binary.sh"
 
-# When running react-native tests, react-native doesn't live in node_modules but in the PROJECT_ROOT
-if [ ! -d "$THIS_DIR/../../../node_modules/react-native-macos" ];
-then
-  PROJECT_ROOT="$THIS_DIR/../../.."
-fi
```

Comment:
Is this technically a diff that we removed this bit?

Reply:
This change is a forward fix: https://github.com/facebook/react-native/pull/36582

Said change never made it into RNCore's 0.72-stable. Its absence doesn't really change anything, since `PROJECT_ROOT` already gets defined as `$THIS_DIR/../../..`. Not sure if this still warrants diff tags though.

Reply:
I think forward fixes I'm ok with not adding diff tags. Thanks!